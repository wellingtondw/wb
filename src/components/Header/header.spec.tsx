import { renderWithTheme } from "../../utils/tests/helper";

import { Header } from ".";

describe("<Header />", () => {
  it("should be able to render correctly", () => {
    const { container } = renderWithTheme(<Header />);

    expect(container.firstChild).toMatchSnapshot();
  });
});
