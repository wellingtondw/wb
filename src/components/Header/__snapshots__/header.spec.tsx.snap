// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Header /> should be able to render correctly 1`] = `
<div
  class="sc-bczRLJ fZgKxy"
>
  <img
    class="sc-gsnTZi kcForN"
    src="webmotors.svg"
  />
</div>
`;
